src/components/Header.jsx

```diff
@@ -1,19 +1,23 @@
 import Link from 'next/link';
 
-import Wrapper from './Positioning/Wrapper';
+import Nav from './Positioning/Nav';
```

Comment:
I am not sure why the folder name is Positioning. For your refactoring i suggetst a name lik "sytles" or "sytled" for Styled Components.